core/src/main/scala/zio/logging/package.scala

```diff
@@ -0,0 +1,14 @@
+package zio
+
+package object logging {
+  def log(line: => String): ZIO[Logger, Nothing, Unit] =
+    ZIO.accessM[Logger](_.log(line))
+
+  def log(level: LogLevel)(line: => String): ZIO[Logger, Nothing, Unit] =
```

Comment:
This only has 3 helper functions. If you trait `Logging`, you can put _all_ helper functions in there, and delegate these three (the very common ones) to those.

That way, importing `zio.logging._` only gives you these 3, but you can access the other 8 or so by using `Logging.XYZ`.

Let me know if this doesn't make sense!

Reply:
all helpers in ```Logging```